airflow/www/templates/airflow/dag.html

```diff
@@ -40,15 +40,26 @@
       <span class="material-icons" aria-hidden="true">keyboard_arrow_up</span>
       DAG: {{ dag.parent_dag.dag_id }}</a>
   {% endif %}
+
   <div>
     <h3 class="pull-left">
       {% if dag.parent_dag is defined and dag.parent_dag %}
         <span class="text-muted">SUBDAG:</span> {{ dag.dag_id }}
       {% else %}
-        <label class="switch-label js-tooltip" title="Pause/Unpause DAG">
-          <input class="switch-input" id="pause_resume" data-dag-id="{{ dag.dag_id }}" type="checkbox" {{ "checked" if not dag.is_paused else "" }}>
-          <span class="switch" aria-hidden="true"></span>
-        </label>
+        {% if appbuilder.sm.can_edit_dag(dag.dag_id) %}
+          <label class="switch-label js-tooltip" title="Pause/Unpause DAG">
+            <input class="switch-input" id="pause_resume" data-dag-id="{{ dag.dag_id }}"
+                  type="checkbox" {{ "checked" if not dag.is_paused else "" }}>
+            <span class="switch" aria-hidden="true"></span>
+          </label>
+        {% else %}
+          <label class="switch-label js-tooltip disabled" title="DAG is {{ 'Paused' if dag.is_paused else 'Active' }}">
+            <input class="switch-input" id="toggle-{{ dag.dag_id }}" data-dag-id="{{ dag.dag_id }}"
+                 type="checkbox" {{ "checked" if not dag.is_paused else "" }} disabled>
+            <span class="switch disabled" aria-hidden="true"></span>
+          </label>
+        {% endif %}
+
```

Comment:
Can keep these simple with just a single set of markup? Paired with my CSS suggestion, a single `disabled` attribute on the `<input>` is all that needs to change.

Would need some extra logic (example below) for the tooltip to change. Alternatively, we could just disable the tooltips too (by removing the `js-tooltip` class).
```suggestion
        {% if appbuilder.sm.can_edit_dag(dag.dag_id) %}
          {% set switch_tooltip = 'Pause/Unpause DAG' %}
        {% else %}
          {% set switch_tooltip = 'DAG is Paused' if dag.is_paused else 'DAG is Active' %}
        {% endif %}
        <label class="switch-label js-tooltip" title="{{ switch_tooltip }}">
          <input class="switch-input" id="pause_resume" data-dag-id="{{ dag.dag_id }}" type="checkbox"{{ " checked" if not dag.is_paused else "" }}{{ " disabled" if not appbuilder.sm.can_edit_dag(dag.dag_id) else "" }}>
          <span class="switch" aria-hidden="true"></span>
        </label>
```